diff --git a/tools/ci_build/build.py b/tools/ci_build/build.py
index 311c58423e..fa3a646895 100644
--- a/tools/ci_build/build.py
+++ b/tools/ci_build/build.py
@@ -368,7 +368,7 @@ def parse_arguments():
     parser.add_argument(
         "--osx_arch",
         default="arm64" if platform.machine() == "arm64" else "x86_64",
-        choices=["arm64", "arm64e", "x86_64", "x86_64;arm64"],
+        choices=["arm64", "arm64e", "x86_64"],
         help="Specify the Target specific architectures for macOS and iOS, This is only supported on MacOS",
     )
     parser.add_argument(
